config/initializers/cors.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins '*'
+    resource '/api/*', headers: :any, methods: %i(get)
```

Comment:
I'm okay with that. It's really going to pay character saving dividends if we add `post`, `patch`, `put` and `delete`. Just think of all those saved `:` and `,`!